Makefile.work

```diff
@@ -294,26 +314,61 @@ endif
 
 endif
 
-DOCKER_BASE_BUILD = docker build --no-cache \
+DOCKER_AUTH:=docker login -u sonicbrcm -p 4b5d1f28-6f43-41da-a794-88805ee8fc2d 
```

Comment:
It's not safe to push to github.

Reply:
Added for the local build, accidentally it got merged into PR code. Just removed it.